### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by romange** [dragonflydb/dragonfly#965] [Announcements]
# Dragonfly v1.0.0

We are thrilled to announce that Dragonfly now supports primary-to-secondary replication! 🎆 🎆  

You can either configure Dragonfly as a secondary instance alongside your existing Redis master,
or establish your own replicated DF-to-DF setup.   The configuration process is quite similar to that of Redis. 
For more information, please check out our [replication setup](https://dragonflydb.io/docs/managing-dragonfly/replication) page.

In addition to this feature, the latest release includes final touches and bug fixes to enhance Dragonfly's stability.
## What's Changed
* fix(tests): Disable logging in replication tests by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/906
* fix(regression-tests): fix cpu print step  by @adiholden in https://github.com/dragonflydb/dragonfly/pull/910
* fix: logging in sentinel_test by @talbii in https://github.com/dragonflydb/dragonfly/pull/869
* fix(pytest): Add master restart delay by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/909
* chore(helm): update golden files to latest version by @Pothulapati in https://github.com/dragonflydb/dragonfly/pull/915
* fix(server): Change WaitForCallbacks mem order by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/914
* fix(tests):Some test cases for redis replication by @ashotland in https://github.com/dragonflydb/dragonfly/pull/918
* fix: send a single RDB_OPCODE_FULLSYNC_END from a snapshot by @romange in https://github.com/dragonflydb/dragonfly/pull/920
* fix(core): Auto cast doubles to ints in lua scripts by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/922
* chore(helm): Make release pipeline to update golden files by @Pothulapati in https://github.com/dragonflydb/dragonfly/pull/916
* feat(server): Implement BITPOS. by @chakaz in https://github.com/dragonflydb/dragonfly/pull/907
* Skip version check if $DFLY_DEV_ENV is set by @ashotland in https://github.com/dragonflydb/dragonfly/pull/928
* fix: Fix a corruption when connection writes interleaving messages by @romange in https://github.com/dragonflydb/dragonfly/pull/926
* feat(workflow): Add Vercel Docs Hook to release workflow by @vahnag in https://github.com/dragonflydb/dragonfly/pull/919
* fix(docs):Delete api_status.md by @ashotland in https://github.com/dragonflydb/dragonfly/pull/921
* feat(server): Pragmas from scripts by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/927
* chore: change thread_local to  __thread when possible. by @romange in https://github.com/dragonflydb/dragonfly/pull/930
* feat: increase expiry period to 8 years. by @romange in https://github.com/dragonflydb/dragonfly/pull/932
* fix(redis_replicaiton_test): fix compare set types by @adiholden in https://github.com/dragonflydb/dragonfly/pull/929
* bug(redis replication): fix multi transaction replication stable state by @adiholden in https://github.com/dragonflydb/dragonfly/pull/934
* fix: correctly set batching mode during pubsub by @romange in https://github.com/dragonflydb/dragonfly/pull/937
* feat(server): hello and info sidekiq compatibility by @ashotland in https://github.com/dragonflydb/dragonfly/pull/941
* feat(ci): Add daily builds BFS verification by @Pothulapati in https://github.com/dragonflydb/dragonfly/pull/936
* fix(replication): redis replication flush all before full sync by @adiholden in https://github.com/dragonflydb/dragonfly/pull/946
* fix(rdb_load): replica load expired keys by @adiholden in https://github.com/dragonflydb/dragonfly/pull/948
* fix(ci): Use correct `x86-64` with builds by @Pothulapati in https://github.com/dragonflydb/dragonfly/pull/950
* Integration test docker image with Go client by @pascaldekloe in https://github.com/dragonflydb/dragonfly/pull/943
* chore: some pytests and logging improvements by @romange in https://github.com/dragonflydb/dragonfly/pull/953
* improve consistency around brpop flow by @romange in https://github.com/dragonflydb/dragonfly/pull/954
* fix: Fix deadlock in the transaction code. by @romange in https://github.com/dragonflydb/dragonfly/pull/956

## New Contributors
* @talbii made their first contribution in https://github.com/dragonflydb/dragonfly/pull/869
* @vahnag made their first contribution in https://github.com/dragonflydb/dragonfly/pull/919
* @pascaldekloe made their first contribution in https://github.com/dragonflydb/dragonfly/pull/943

**Full Changelog**: https://github.com/dragonflydb/dragonfly/compare/v0.17.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dragonflydb/dragonfly/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by Tashitasha** [comment]
Hi
